state/aerospike/aerospike.go

```diff
@@ -171,7 +173,7 @@ func (aspike *Aerospike) Get(req *state.GetRequest) (*state.GetResponse, error)
 
 	return &state.GetResponse{
 		Data: value,
-		ETag: fmt.Sprintf("%d", record.Generation),
+		ETag: ptr.String(strconv.FormatUint(uint64(record.Generation), 10)),
```

Comment:
Why this is better than sprintf?

Reply:
It's not a matter of being better, just that this simple case does not require it. Unless you need the the formatting options that Sprintf offers (i.e. padding, left justification, multiple value placeholders), it is inefficient compared to using the `strconv` package. IMO linters should pick this up as it is very similar to [using `fmt.Sprintf("%s", str)` unnecessarily](https://staticcheck.io/docs/checks#S1025) or [unnecessary use of fmt.Sprint](https://staticcheck.io/docs/checks#S1039).

For comparison:
[Go's formatting code](https://github.com/golang/go/blob/2b50ab2aee75d3c361fcd1eb39e830e2e73056b6/src/fmt/print.go#L974-L1147)
[Go's integer conversion code](https://github.com/golang/go/blob/2b50ab2aee75d3c361fcd1eb39e830e2e73056b6/src/strconv/itoa.go#L105-L152)